### ISSUE TITLE
Zenodo or similar for Citation?

### ISSUE THREAD

**Issue by vsoch** [containers/podman#9654]
Hey Podman maintainers!

If someone wanted to cite podman, how should they go about it? For example, if there is an associated publication we could use that, or a record on Zenodo. (https://zenodo.org). It indeed is a very academic sort of thing, but in the case that someone does wnat to cite it, I thought it might be nice to have the reference. Thank you!

**Comment by rhatdan** [comment]
It would probably be best if you suggested something, or showed us a similar open source project that was "cited".  Bottom line, I have no idea how to answer this.

**Comment by vsoch** [comment]
Sure! Singularity had a Zenodo DOI before we published the first paper (see here https://github.com/hpcng/singularity#citing-singularity) and the zenodo page here: https://zenodo.org/record/4435194#.YEZa4P5lBH4. It is linked to releases and automatically updates with each one. Importantly, it provides a DOI (digital object identifier) that links to a specific release.

All papers in JOSS (The Journal of open source software) are issued Zenodo DOIs: https://joss.theoj.org/, and you can read about why it's important to cite software [here is an example](https://www.software.ac.uk/how-cite-software).

In short, it's about 5 minutes of work to connect a repository and then (if you want) you can add a badge that links to the DOI. There is no work beyond that because it updates with releases. The benefit/pro is that you provide this DOI for anyone using your software to easily cite. I don't think there are any compelling cons, unless you decide that the work is too much and you don't want to support anyone present or future that wants to cite the software. Let me know if you have any other questions!

**Comment by vsoch** [comment]
Here is a quick example relevant to podman. A researcher did a comparison study between container technologies (Podman being one) https://www.diva-portal.org/smash/record.jsf?pid=diva2%3A1450777&dswid=7004 and because there is no citation provided for Podman, they used a blog post:

![image](https://user-images.githubusercontent.com/814322/110358550-c85c2b80-7ff9-11eb-90fd-63a667afd324.png)

That could tell you generally the date that they referenced the page, but it doesn't link to the actual commit / version of the software. Someone trying to reproduce this study could guess at best. Here is another paper focused on Podman:

https://link.springer.com/chapter/10.1007/978-3-030-59851-8_23

it's worse because they just reference podman.io.

![image](https://user-images.githubusercontent.com/814322/110358848-0a856d00-7ffa-11eb-84d8-ea1566834021.png)

And a third, which doesn't even attempt to cite it (but it's a part of their study!)

https://sc20.supercomputing.org/proceedings/sotp/sotp_files/sotp120s2-file1.pdf

So - in short, when people write these papers they will come looking here first for a link / DOI to cite in the repository. When they can't find that, they fallback to documentation and web sites, which aren't really useful for going back, installing the version of podman for which the work was done, and using it (which is the basis of scientific reproducibility). So the need is there, as evidenced by these kinds of papers (there are more but I didn't look beyond the first page of search results).

**Comment by rhatdan** [comment]
@vrothberg @fatherlinux @TomSweeneyRedHat @baude WDYT?

**Comment by baude** [comment]
looks ok to me

**Comment by TomSweeneyRedHat** [comment]
I'm late to the party, but it SGTM.  @fatherlinux WDYT?

**Comment by fatherlinux** [comment]
SGTM. I think it makes sense to have that.

**Comment by TomSweeneyRedHat** [comment]
@vsoch I created it, probably needs modification, any suggestions for: https://zenodo.org/record/4735634  ?

**Comment by vsoch** [comment]
It looks great! My one suggestion would be to change "version X and beyond!" (paging Buzz Lightyear!) to just be the latest version, e.g., this part:

![image](https://user-images.githubusercontent.com/814322/116947807-e9787b80-ac3a-11eb-976b-d869aa88fd87.png)

The way Zenodo works is that it's linked to your releases here, so that DOI will always link to the latest. And if someone using a prior version wants to cite, they can still use it okay. I think we would just want that version to match the version here.

Once it's ready, I can help to share that Podman has a DOI! I've come across several publications that "cite" it with the podman.io address, so there are definitely people out there that would benefit from knowing about it.

**Comment by vsoch** [comment]
Also does the version there link up to the one in GitHub releases? I'm seeing v3.1.2 https://github.com/containers/podman/releases

**Comment by vsoch** [comment]
Ping! I think we were almost done but I wanted to discuss some possible tweaks.

**Comment by vsoch** [comment]
I guess it's okay, fine to close.
